configuration/globalproperties/indexTesting.xml

```diff
@@ -0,0 +1,95 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.indexPatients</property>
+      <value>
+      select
+        distinct p.uuid as uuid,
+        concat(pn.given_name," ", ifnull(pn.family_name,'')) as 'PROGRAM_MODULE_LABEL_NAME_KEY',
+        getPatientPhoneNumber(p.person_id) as 'PROGRAM_MODULE_LABEL_PHONE_NUMBER_KEY',
+        getPatientAge(p.person_id) as 'PROGRAM_MODULE_LABEL_AGE_KEY',
+        p.gender as 'PROGRAM_MODULE_LABEL_GENDER_KEY',
+        getNumberOfContactsRelatedToIndex(p.person_id) as 'PROGRAM_MODULE_LABEL_NR_CONTACTS_KEY',
+        concat(
+          getNumberOfEnrolledContactsRelatedToIndex(p.person_id),
+          "/",
+          getNumberOfContactsRelatedToIndex(p.person_id)) as 'PROGRAM_MODULE_LABEL_NR_ENROL_CONTACTS_KEY'
+      from
+        person p join person_name pn on p.person_id = pn.person_id and pn.voided = 0 and p.voided=0
+      where 
+        patientIsIndex(p.person_id) AND
+        (getNumberOfContactsRelatedToIndex(p.person_id) = 0 or
+        getNumberOfEnrolledContactsRelatedToIndex(p.person_id) < getNumberOfContactsRelatedToIndex(p.person_id))
+      order by p.date_created desc;
+      </value>
+    </globalProperty>
+    <globalProperty>
+      <property>emrapi.sqlSearch.indexContacts</property>
+      <value>
+      select
+        distinct pi.identifier as ID,
+        concat("",p.uuid) as uuid,
+        concat(pn.given_name," ", ifnull(pn.family_name,'')) as 'PROGRAM_MODULE_LABEL_NAME_KEY',
+        getIndexNames(p.person_id) as 'PROGRAM_MODULE_LABEL_INDEX_NAME_KEY',
+        getIndexRelationships(p.person_id) as 'PROGRAM_MODULE_LABEL_INDEX_RELATIONSHIP_KEY',
+        getPatientPhoneNumber(p.person_id) as 'PROGRAM_MODULE_LABEL_PHONE_NUMBER_KEY',
+        getPatientAge(p.person_id) as 'PROGRAM_MODULE_LABEL_AGE_KEY',
+        p.gender as 'PROGRAM_MODULE_LABEL_GENDER_KEY',
+        getPatientMostRecentProgramAttributeCodedValueFromName(p.person_id, 'PROGRAM_MANAGEMENT_1_NOTIFICATION_METHOD', ${language}) as 'PROGRAM_MODULE_LABEL_NOTIFICATION_METHOD_KEY',
+        getPatientMostRecentProgramAttributeValueFromName(p.person_id, 'PROGRAM_MANAGEMENT_2_NOTIFICATION_DATE') as 'PROGRAM_MODULE_LABEL_NOTIFICATION_DATE_KEY',
+        getPatientMostRecentProgramAttributeCodedValueFromName(p.person_id, 'PROGRAM_MANAGEMENT_3_NOTIFICATION_OUTCOME', ${language}) as 'PROGRAM_MODULE_LABEL_NOTIFICATION_OUTCOME_KEY',
+        getMostRecentDateObservation(p.person_id, 'HIV Test Date') as 'PROGRAM_MODULE_LABEL_TEST_DATE_KEY',
+        getMostRecentCodedObservation(p.person_id, 'Services', ${language}) as 'PROGRAM_MODULE_LABEL_TEST_PLACE_KEY',
```

Comment:
@tmvumbi2 The correct Service/Location selected can be obtained from the encounter location for the test capture.